### ISSUE TITLE
How can I move a file to another directory?

### ISSUE THREAD

**Issue by Kurren123** [stevearc/oil.nvim#460]
Is it possible to move a file from its current directory to a different one using oil.nvim?

I know we can move it to a subdirectory by putting `NewDir/` in front of the file name. But moving it out of the current directory will require printing the file path relative to a root folder.

**Comment by TheLeoP** [comment]
1. Cut the file from the current directory by putting the cursor above it and `dd`
2. Navigate to the target directory (using `telescope`/`fzf-lua`, `:e`, or the builtin `-` keymap to go up a directory)
3. Paste the file `p`
4. Save the buffer `:w`

**Comment by Kurren123** [comment]
Thank you!

**Comment by sbushmanov** [comment]
What if I'm pasting between 2 nvim instances opened on tmux?

I'm getting error on write:

```
Error detected while processing BufWriteCmd Autocomma
nds for "oil://*":
Error parsing oil buffers
```

**Comment by TheLeoP** [comment]
Oil keeps track of each file by assigning an ID to it. The ID is local to each neovide instance. So, you can't do file operations betweens different Neovim instances

**Comment by sbushmanov** [comment]
@TheLeoP 
Thanks, got it.
Can you suggest then how to paste a file to the desired directory in the `Telescope` window?

![image](https://github.com/user-attachments/assets/ff7ce8b0-044f-4b43-92a9-81da2cf69d74)

**Comment by windowsrefund** [comment]
From an efficiency perspective, that just seems like an odd (wrong) thing to want to do? Strikes me as...

1. I just entered my house through the front door and am standing in the lobby.
2. I see the kitchen is straight ahead and realize I want a beer from the fridge.
3. I turn around, exit through the front door I just entered.
4. I walk around the house and enter through the back patio door and proceed toward the fridge.

In fact, that may have been generous as we may actually be talking about breaking a window.
